release/mixer.sh

```diff
@@ -20,9 +20,16 @@ SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
 var_load_all
 
 fetch_bundles() {
-    log_line "Fetching downstream bundles:"
+    if "$IS_UPSTREAM"; then
+        log_line "Fetching bundles:"
+    else
+        log_line "Fetching downstream bundles:"
+    fi
     rm -rf ./local-bundles
-    git clone --quiet "${BUNDLES_REPO}" local-bundles
+    local temp_bundles="$(mktemp -d)"
```

Comment:
Give preference to use the work_dir to hold temporary artefacts, it is "guarantee" to be a large storage device.